astro/src/content/docs/release-notes/index.mdx

```diff
@@ -45,6 +45,50 @@ import { YouTube } from '@astro-community/astro-embed-youtube';
 
 Looking for release notes older than 1.44.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications) <span class="not-prose no-underline"><a class="ml-2" href="/docs/releases.xml"><i class="fas fa-xs fa-rss text-orange-700 text-2xl" width="50px" /></a></span>
 
+<ReleaseNoteHeading version='1.57.0' releaseDate='April 4, 2025' name='Webhook Wombat'/>
+
+<GeneralMigrationWarning>
+  This release provides a handful of improvements to the webhook event log. In versions 1.53.0 through 1.56.0, very large webhook event log volumes could impact performance during log cleaning operations. 
+
+  If you are currently using a version of FusionAuth earlier than 1.53.0, we recommend upgrading to version 1.57.0 and not an intermediate version.
```

Comment:
Not sure if we should be quite so strong in the wording here. I think it's acceptable and maybe a good idea to go through intermediate versions, but they should continue on to 1.57.0 as quickly as possible. Maybe it's better not to muddy the waters and just leave it as is.